classes/web_processor.php

```diff
@@ -41,6 +41,7 @@ class web_processor implements processor {
      * @param manager $manager The profiler manager object
      */
     public function init(manager $manager) {
+        global $USER;
```

Comment:
This does not appear to be used anywhere in the method, please consider removing it.